app/helpers/shared_helper/ab_list_helper.rb

```diff
@@ -1,135 +1,25 @@
 module SharedHelper::AbListHelper
   private
 
-  def shared_button_summary(record, sb_items, resolve, custom_button)
+  def shared_button_list_summary(record)
     summary = [
-      shared_button_basic_info(record, sb_items, custom_button),
-      object_details(record, resolve),
-      object_attribute(record, resolve),
-      visibility_expression(record),
-      attribute_value_pair(resolve),
-      visibility(custom_button, sb_items)
+      shared_button_list_basic_info(record)
     ]
     safe_join(summary)
   end
 
-  def shared_button_basic_info(record, sb_items, custom_button)
-    style = record&.options&.[](:button_color)&.to_s
-    display_for = record&.options&.[](:display_for)
-    display_how_map = {'single' => _('Single entity'), 'list' => _('List'), 'both' => _('Single and list')}
-    display_for_value = display_how_map[display_for] if display_for.present?
-
-    submit_how = record&.options&.[](:submit_how)
-
-    submit_how_map = {'all' => _('Submit all'), 'one' => _('One by one')}
-    submit_for_value = submit_how_map[submit_how] if submit_how.present?
-    rows = [
-      row_data(_('Button Type'), record&.options&.[](:button_type)&.capitalize),
-      row_data(_('Name'), record&.name),
-      row_data(_('Display on button'), {:input => "checkbox", :name => "display", :checked => true, :disabled => true, :label => ''}),
-      row_data(_('Description'), record&.description),
-      row_data(_('Image'), record&.options&.[](:button_icon), style, :icon => true),
-      row_data(_('Dialog'), sb_items[:dialog_label]),
-      row_data(_('Open URL'), record&.options&.[](:open_url)),
-      row_data(_('Display for'), display_for_value&.capitalize),
-      row_data(_('Submit'), submit_for_value&.capitalize),
-    ]
-    service_template_name = custom_button.uri_attributes[:service_template_name]
-    uri_attributes_hosts = custom_button.uri_attributes[:hosts]
-    rows.insert(2, row_data(_('Ansible Playbook'), service_template_name)) if service_template_name.present?
-    rows.insert(3, row_data(_('Target'), uri_attributes_hosts)) if uri_attributes_hosts.present?
-
-    miq_structured_list({
-                          :title => _('Basic Information'),
-                          :mode  => _('run_button_summary'),
-                          :rows  => rows
-                        })
-  end
-
-  def object_details(_custom_button, resolve)
-    data = {
-      :title => _('Object Details'),
-      :mode  => _('object_details'),
-      :rows  => [
-        row_data(_('System/Process/'), resolve[:new][:instance_name]),
-        row_data(_('Message'), resolve[:new][:object_message]),
-        row_data(_('Request'), resolve[:new][:object_request])
-      ]
-    }
-    miq_structured_list(data)
-  end
-
-  def object_attribute(record, resolve)
-    record&.enablement_expression
+  def shared_button_list_basic_info(record)
+    style = record.set_data.key?(:button_color) ? "color: #{record.set_data[:button_color]};" : nil
```

Comment:
please remove the ; for this line
`style = record.set_data.key?(:button_color) ? "color: #{record.set_data[:button_color]}" : nil`